src/components/setting-item/SettingItem.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Box } from '@mui/material'
+import Typography from '@mui/material/Typography'
+import Switch from '@mui/material/Switch'
+
+import { defaultStyles } from '~/components/app-content-switcher/AppContentSwitcher.styles'
+import { styles } from '~/components/setting-item/SettingItem.styles'
+
+interface SettingItemProps {
+  title: string
+  subtitle: string
+  checked: boolean
+  dataTestId: string
+  onChange: (
+    event: React.ChangeEvent<HTMLInputElement>,
+    checked: boolean
+  ) => void
+}
+
+const SettingItem: React.FC<SettingItemProps> = ({
+  title,
+  subtitle,
+  checked,
+  dataTestId,
```

Comment:
maybe you should set a default value for dataTestId prop

Reply:
I made it optional